test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -836,7 +839,6 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
             test.form.setUsername('testuser');
             test.form.setPassword('pass');
             test.form.submit();
-            expect(test.beacon.isLoadingBeacon()).toBe(true);
```

Comment:
I think it's fine to remove this line - end state is *mostly* what we care about